app/views/criteria/_boot.js.erb

```diff
@@ -28,4 +28,41 @@
       });
     }
   });
+  function add_level(id) {
+    var rubric_form = $('#' + id);
```

Comment:
Use `let` instead of `var` (in general, we're phasing out the `var`s`).

Reply:
should I be using const for the variables that make sense? Like should rubric_form which would not need to be reassigned?